pwnedhub/static/react/messages.js

```diff
@@ -143,14 +143,16 @@ var MessageDelete = React.createClass({
 });
 
 var Message = React.createClass({
-    render() {
+    getMessageStyle() {
         // set an inline style if the message is owned by the current user
-        var messageStyle = {};
         if (this.props.message.is_owner == true) {
```

Comment:
I elected to go with a ternary operator here.

`return this.props.message.is_owner == true ? {fontWeight: "bold"} : {}`